src/Widgets/DisplayWindow.vala

```diff
@@ -19,13 +19,16 @@
  * Authored by: Corentin Noël <corentin@elementary.io>
  */
 
-public class Display.DisplayWindow : Gtk.Window {
+public class Display.DisplayWindow : Hdy.ApplicationWindow {
+    private const int _leftMargin = 15;
+    private const int _topMargin = 45;
```

Comment:
I see what you're doing with the two different margins to account for the panel, but then this looks off on displays without a panel. And I think I see why @danrabbit might have been going for a pill-shape since using a standard 12px margin keeps the top of this window against the bottom of the panel, but we could look at that in a follow-up branch. For now, I say let's use a standard margin size and either deal with the pill shape or accounting for panel struts in a follow-up:

```suggestion
    private const int spacing = 12;
```

Reply:
I`ve tried with the same spacing for both left and up, but ended up being annoyed by the label touching the panel, then on other monitors it did not. But i agree with the change.